src/Extend/Frontend.php

```diff
@@ -51,10 +54,22 @@ public function route($path, $name, $content = null)
         return $this;
     }
 
+    /**
+     * @param callable|string $callback
+     * @return $this
+     */
+    public function document($callback)
+    {
+        $this->documentCallback = $callback;
```

Comment:
I think we should allow multiple document callbacks to be registered on a single extender. There is loads of stuff that can be done to HtmlDocument so people might want to organise it into separate callbacks/classes:

```php
(new Extend\Frontend('forum'))
    ->document(AddFooter::class)
    ->document(AddSeoStuff:class)
```

Reply:
In one extension?

Reply:
Yeah

Reply:
Hmm I wonder if `content` might be a better name for the method for consistency with the internal API. And you could either pass a callback, or a class name which implements `ContentInterface`

Reply:
I took care of the multiple calls, will do the rest tomorrow. :zzz: